src/core_atmosphere/physics/physics_noaa/SMOKE/mpas_smoke_init.F90

```diff
@@ -0,0 +1,91 @@
+!>\file  mpas_smoke_init.F90
+!! This file initializes and saves the chemistry pointers so they are available to lower routines
+!
+! Jordan.Schnell@noaa.gov  
+! 01/2025
+!
+module mpas_smoke_init
+
+  use mpas_kind_types
+  
+  implicit none
+
+   integer, save ::  &
+   p_smoke_fine, &
+   p_smoke_coarse, &
+   p_dust_fine,  &
+   p_dust_coarse, &
+   p_polp_all, &
+   p_polp_tree, &
+   p_polp_grass  , & 
+   p_polp_weed, &
+   p_pols_tree, &
+   p_pols_grass  , & 
+   p_pols_weed, &
+   p_pols_all, &
+   p_unspc_fine, &
+   p_unspc_coarse, &
+   p_ssalt_fine, &
+   p_ssalt_coarse, &
+   p_no3_a_fine, &
+   p_so4_a_fine, &
+   p_nh4_a_fine, &
+   p_so2, &
+   p_nh3, &
+   p_mold_fine, &
+   p_bact_fine, &
+   p_plastic_fine, &
+   p_dms
+
+   contains
+
+  subroutine set_scalar_indices( chemistry_start,                                     &
+                                 index_smoke_fine) !, index_smoke_coarse,                &
```

Comment:
it's a bit confusing to have all these comments

Reply:
The commented lines are for the two versions (MPAS-Aerosol and MPAS-Smoke/Dust).